packages/vm/lib/index.ts

```diff
@@ -98,15 +98,35 @@ export interface VMOpts {
  * This class is an AsyncEventEmitter, please consult the README to learn how to use it.
  */
 export default class VM extends AsyncEventEmitter {
-  opts: VMOpts
-  _common: Common
+  
+  /**
+   * The StateManager used by the VM
+   */
   stateManager: StateManager
```

Comment:
Shouldn't `stateManager`, `blockchain`, 'common' be marked `readonly`?

Reply:
Yes, good point, have added in e50af8003575ed7bb2b82b3a68de8a7e60354fd4. Realized that I actually had a wrong understanding of the [Readonly modifier](https://www.typescriptlang.org/docs/handbook/classes.html).